chimera_app/server.py

```diff
@@ -268,7 +268,7 @@ def shortcut_create():
         ext = get_ext(image_urls[img_type])
         image_paths[img_type] = os.path.join(BANNER_DIR, img_type, platform, f"{name}{ext}")
         ensure_directory_for_file(image_paths[img_type])
-        download = requests.get(image_urls[img_type])
+        download = requests.get(image_urls[img_type], timeout=5)
```

Comment:
Do you know if this timeout is for the total time to download or just to initial connection?
I could imagine some image files taking longer than 5 seconds to download.

Reply:
Should be a relatively fast transaction. The timeout is for both connect and read, so the whole thing will timeout in at most double that: 5 seconds to connect, 5 seconds to read. So maybe put it to a higher value?